toolchain/check/import_ref.cpp

```diff
@@ -136,9 +142,51 @@ class ImportRefResolver {
     const auto& param_refs = import_ir_.inst_blocks().Get(param_refs_id);
     llvm::SmallVector<SemIR::InstId> new_param_refs;
     for (auto [ref_id, const_id] : llvm::zip(param_refs, const_ids)) {
-      new_param_refs.push_back(context_.AddInstInNoBlock(
-          {SemIR::ImportRefUsed{context_.GetTypeIdForTypeConstant(const_id),
-                                import_ir_id_, ref_id}}));
+      // Figure out the param structure. This echoes
+      // Function::GetParamFromParamRefId.
```

Comment:
Not blocking at all, but curious if you have anything in mind to avoid the parallel code structure?